run/tv/Makefile

```diff
@@ -0,0 +1,43 @@
+bmv2_p4c_out := ${ONOS_ROOT}/pipelines/fabric/impl/src/main/resources/p4c-out
+tofino_p4c_out := ${FABRIC_TOFINO}/src/main/resources/p4c-out
+
+define run_tests
+$(if $(filter $(DEVICE),tofino),
+python -u ptf_runner.py --device $(DEVICE) --port-map $(PORTMAP) \
+	--ptf-dir fabric.ptf --cpu-port 320 --device-id 1 \
+	--grpc-addr $(GRPCADDR) \
+	--p4info $(tofino_p4c_out)/$(1)/tofino/mavericks_sde_9_0_0/p4info.txt \
```

Comment:
This implementation will only work for mavericks and compiler output for SDE 9.0.0. Is there a way to make this parametric? I think simply allowing to pass the full location of the compiler output should work. I.e., setting `$(tofino_p4c_out)` to `/$(1)/tofino/mavericks_sde_9_0_0/`